man/influxd-restore.txt

```diff
@@ -3,32 +3,71 @@ influxd-restore(1)
 
 NAME
 ----
-influxd-restore - Restores the metastore, databases, retention policies, or specific shards
+influxd-restore - Restores the metastore, databases, retention policies, or specific shards.  Legacy mode requires
+the instance to be stopped before running, and will wipe all databases from the system (e.g., for disaster recovery).
+The improved online mode requires the instance to be running, and the database name used must not already exist.
 
 SYNOPSIS
 --------
-'influxd restore' [options] PATH
+'influxd restore' [-legacy] [flags] PATH
 
 DESCRIPTION
 -----------
 Uses backups from the PATH to restore the metastore, databases, retention policies, or specific shards. The InfluxDB process must not be running during a restore.
 
 OPTIONS
 -------
+Legacy mode consumes files in the pre-1.5 file format. PATH is a directory containing the backup data
+
+Legacy options
+
 -metadir <path>::
-  If set, the metastore will be recovered to the given path. Optional.
+            Optional. If set the metastore will be recovered to the given path.
 
 -datadir <path>::
-  If set, the restore process will recover the specified database, retention policy, or shard to the given directory. Optional.
+            Optional. If set the restore process will recover the specified
+            destinationDatabase, retention policy or shard to the given directory.
 
 -database <name>::
-  Will restore the database TSM files. Required if no metadir is given. Optional.
+            Optional. Required if no metadir given. Will restore the database's
+            TSM files.
```

Comment:
Just say "a single database's data". TSM files are an implementation detail of the current storage engine.